### PR TITLE
Add tippecanoe to conda-forge

### PR DESCRIPTION
**Pull request by adbeda**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by adbeda** [comment]
tippecanoe is not supported for windows as explained [here (https://github.com/mapbox/tippecanoe/issues/712#issuecomment-460363603)

File: recipes/tippecanoe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+source:
+    url: https://github.com/mapbox//{{name}}/archive/{{ version }}.tar.gz
+    sha256: 7a2dd2376a93d66a82c8253a46dbfcab3eaaaaca7bf503388167b9ee251bee54
+
+build:
+  skip: true   # [win]
+  number: 0
+  script:
+    - ls
+    - make -j
```
Comment (marcelotrevisani) [inline recipes/tippecanoe/meta.yaml]: I think here it should be
```suggestion
    - make -j$CPU_COUNT
```

File: recipes/tippecanoe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    - make install
+
+requirements:
+  build:
+    - make
+    - sqlite
+    - zlib
```
Comment (marcelotrevisani) [inline recipes/tippecanoe/meta.yaml]: ``sqlite`` and ``zlib`` should be in the ``host`` section

**Comment by adbeda** [comment]
@conda-forge/help-c-cpp 
when adding test commands to check the help information on this package with `tippecanoe -- h` the test prints out results as expected but the build fails and the package is eventually moved to the broken folder. any idea what might be causing this?

File: recipes/tippecanoe/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    - {{ compiler ('c') }}
+    - {{ compiler ('cxx') }}
+  host:
+    - sqlite
+    - zlib
+
+test:
+  commands:
+    - which tippecanoe
+    - tippecanoe -- help
```
Comment (xhochy) [inline recipes/tippecanoe/meta.yaml]: A space too much? This command is failing the build.
```suggestion
    - tippecanoe --help
```

Comment (adbeda) [reply recipes/tippecanoe/meta.yaml]: thanks for catching that. it will most likely drive me crazy if thats the only issue causing it to fail.

Comment (adbeda) [reply recipes/tippecanoe/meta.yaml]: @xhocky. yup. it did work

**Pull request merged**